src/actions/users.js

```diff
@@ -201,6 +201,25 @@ export function getProfiles(page = 0, perPage = General.PROFILE_CHUNK_SIZE) {
     };
 }
 
+export function getMissingProfilesByIds(userIds) {
```

Comment:
nice!! there are a few places where we do this all the time, and we do it also for statuses like for `getProfilesAndStatusesForMembers` in `/src/actions/teams.js`